BEP2.md

```diff
@@ -0,0 +1,207 @@
+# BEP-2: Tokens on Binance Chain
+
+
+- [BEP-2: Tokens on Binance Chain](#bep-2--tokens-on-binance-chain)
+  * [1.  Summary](#1--summary)
+  * [2.  Abstract](#2--abstract)
+  * [3.  Motivation](#3--motivation)
+  * [4.  Specification](#4--specification)
+    + [4.1  Native Token on Binance Chain: BNB](#41--native-token-on-binance-chain--bnb)
+    + [4.2 Token Properties](#42-token-properties)
+    + [4.3 Token Management Operation](#43-token-management-operation)
+      - [4.3.1  Issue token](#431--issue-token)
+      - [4.3.2  Transfer Tokens](#432--transfer-tokens)
+      - [4.3.3  Freeze Tokens](#433--freeze-tokens)
+      - [4.3.4  Unfreeze Tokens](#434--unfreeze-tokens)
+      - [4.3.5 Mint Tokens](#435-mint-tokens)
+      - [4.3.6 Burn Tokens](#436-burn-tokens)
+  * [5. License](#5-license)
+
+
+
+
+## 1.  Summary
+
+This BEP describes a proposal for token management on the Binance Chain.
+
+## 2.  Abstract
+
+BEP-2 Proposal describes a common set of rules for token management within the Binance Chain ecosystem. It introduces the following details of a token on Binance Chain:
+
+- What information makes a token on Binance Chain
+- What actions can be performed on a token on Binance Chain
+
+## 3.  Motivation
+
+Design and issue asset on the Binance Chain, as the basic economic foundations of the blockchain.
+
+## 4.  Specification
+
+### 4.1  Native Token on Binance Chain: BNB
+
+The Binance Token, BNB, is the native asset on Binance Chain and created within Genesis Block. BNB was an ERC20 token, but after Binance Chain is live, all BNB ERC20 tokens are swapped for BNB token on Binance Chain. All users who hold BNB ERC20 tokens can deposit them to Binance.com, and upon withdrawal, the new Binance Chain native tokens will be sent to their new addresses.
```

Comment:
add as 2nd sentence: `As the native asset, BNB would be used for fees (gas) and staking on Binance Chain.`